Cargo.toml

```
@@ -31,7 +32,7 @@ default = ["std"]
 
 # Enables all features, including support of simplified Cocoon API, using `rand::thread_rng`,
 # and API related to `std::io`: wrap to writer, unwrap from reader.
-std = ["alloc", "rand/std"]
+std = ["alloc", "rand/std", "thiserror"]
```

Let's make it optional, you will enable it in your code.

```suggestion
std = ["alloc", "rand/std"]
```

Then I can quickly merge it, I will improve the documentation and I will release it shortly.

---

@madonuko Broke your PR... See the fix: https://github.com/fadeevab/cocoon/pull/28

What do you think?

---

Looks pretty much perfect to me